docs/deployment/caching/profiles.md

```diff
@@ -0,0 +1,116 @@
+---
+myst:
+  html_meta:
+    "description": "Caching profiles for Plone"
+    "property=og:description": "Caching profiles for Plone"
+    "property=og:title": "Caching profiles for Plone"
+    "keywords": "Plone, deployment, automation, caching"
+---
+
+(caching-profiles-label)=
+
+# Caching profiles
+
+All persistent configuration for the caching machinery is stored in the configuration registry, as managed by `plone.app.registry`.
+
+This can be modified using the `registry.xml` GenericSetup import step.
+
+The *Import settings* tab of the control panel allows you to import these caching profiles.
+
+
+## Default caching profiles
+
+`plone.app.caching` includes three default caching profiles.
+
+Two of these  profiles encapsulate the cache settings that are known to work well with a typical default Plone installation:
+
+### Without caching proxy
+
+Settings useful for setups without a caching proxy.
+
+### With caching proxy
+
+Settings useful for setups with a caching proxy such as Squid or Varnish.
+The only difference from the "without caching proxy" profile are some settings to enable proxy caching of files/images in content space and content feeds.
```

Comment:
I am pretty sure this is not true anymore.

Reply:
I just checked the profiles, and it is it. Feed and Files use `moderateCaching` with `smaxage`:

## With Proxy
```xml
  <!-- plone.content.file -->
  <record name="plone.app.caching.moderateCaching.plone.content.file.smaxage">
    <field ref="plone.app.caching.moderateCaching.smaxage" />
    <value>86400</value>
  </record>
  <record name="plone.app.caching.moderateCaching.plone.content.file.lastModified">
    <field ref="plone.app.caching.moderateCaching.lastModified" />
    <value>True</value>
  </record>
```

## Without Proxy
```xml
  <record name="plone.app.caching.weakCaching.plone.content.file.lastModified">
    <field ref="plone.app.caching.weakCaching.lastModified" />
    <value>True</value>
  </record>
```